hvantk/datasets/dataset_creation_registry.py

```
@@ -0,0 +1,384 @@
+            if Path(self.registry_file).exists():
+                with open(self.registry_file, 'r') as f:
+                    data = json.load(f)
+                    self.datasets = {
+                        dataset_id: self._dict_to_dataset_entry(entry_data)
+                        for dataset_id, entry_data in data.items()
+                    }
+                logger.info(f"Loaded {len(self.datasets)} datasets from registry")
+            else:
+                logger.info("Registry file not found, starting with empty registry")
+                self.datasets = {}
+        except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)